content/en/dora_metrics/failures/incident_api.md

```diff
@@ -72,22 +74,22 @@ curl -X POST "https://api.{{< region-param key="dd_site" >}}/api/v2/dora/inciden
 EOF
 ```
 
-## Change Failure Rate
+## Calculating change failure rate
 
-The Change Failure Rate metric is calculated as the percentage of incident events out of the total number of deployments.
-Send both [deployment events][2] and [incident events](#overview) to correctly populate this metric.
+The change failure rate metric is calculated as the percentage of incident events out of the total number of deployments.
 
-## Time to restore service
+To populate this metric, send your [deployment events][2] and [incident events](#overview) to Datadog.
```

Comment:
I think we are giving the description of how the metrics are calculated in several places, which might be confusing for customers. Could we have a small paragraph in the main page mentioning how each metric is computed instead, as it's independent from the source?

Reply:
yes - thanks for pointing this out!!